lib/variable/test/to_unit_test.cpp

```diff
@@ -274,3 +274,19 @@ TEST(ToUnitTest, small_to_large_rounding_error_double) {
                     units::Unit("m")),
             one_m);
 }
+
+TEST(ToUnitTest, small_number_to_small_unit) {
```

Comment:
Did the problem (running out of range with int64) arise only with small number or also with large numbers?